pallets/xcm-helper/src/lib.rs

```diff
@@ -65,6 +72,11 @@ pub mod pallet {
         type BlockNumberProvider: BlockNumberProvider<BlockNumber = Self::BlockNumber>;
     }
 
+    /// Total amount of charged assets to be used as xcm fees.
+    #[pallet::storage]
+    #[pallet::getter(fn insurance_pool)]
+    pub type InsurancePool<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
```

Comment:
shouldn't be moved here because it's liquid staking specific

Reply:
replied above